libs/shared/components/src/auth/UserMenu/userProfile.graphql.ts

```diff
@@ -8,3 +8,11 @@ export const USER_PROFILE = gql`
     }
   }
 `
+
+export const UPDATE_USER_PROFILE = gql`
+  mutation updateProfile($input: UpdateUserProfileInput!) {
+    updateProfile(input: $input) {
+      locale
```

Comment:
You should always query the ID field of the object you're mutating ... that way, apollo knows how to update related cached objects.

In this case, nationalId should be queried as well.

Reply:
Ah yes, thanks.